tests/s3/copy_object/test_parallel_copy_object.py

```diff
@@ -365,3 +381,140 @@ def test_44793(self):
+                                      f"{obj_type}-obj3"))
+            resp = self.s3_obj.object_info("dest-bucket1", f"{obj_type}-obj2")
+            assert_utils.assert_equal(resp1[1]["ETag"], resp[1]["ETag"])
+            resp = self.s3_obj.object_info("dest-bucket2", f"{obj_type}-obj3")
+            assert_utils.assert_equal(resp1[1]["ETag"], resp[1]["ETag"])
+            self.parallel_copy_object(("src-bucket1", f"{obj_type}-obj1", "dest-bucket1",
+                                      f"{obj_type}-obj2"),
+                                      ("src-bucket1", f"{obj_type}-obj1", "dest-bucket2",
+                                      f"{obj_type}-obj2"))
+            resp = self.s3_obj.object_info("dest-bucket1", f"{obj_type}-obj2")
+            assert_utils.assert_equal(resp1[1]["ETag"], resp[1]["ETag"])
+            resp = self.s3_obj.object_info("dest-bucket2", f"{obj_type}-obj2")
+            assert_utils.assert_equal(resp1[1]["ETag"], resp[1]["ETag"])
+        LOGGER.info("ENDED: Test Parallel copy from single source buckets to one or multiple "
+                    "destination buckets (simple and multipart source objects)")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.s3_object_copy
+    @pytest.mark.tags("TEST-44804")
+    @CTFailOn(error_handler)
+    def test_44804(self):
+        """Test Reverse copy operations"""
+        LOGGER.info("STARTED: Test Reverse copy operations (simple and multipart source objects)")
+        for obj_type in ["simple", "multipart"]:
+            if obj_type == "simple":
+                resp1 = self.create_put_object("src-bucket1", f"{obj_type}-obj1", 100)
+                resp2 = self.create_put_object("dest-bucket1", f"{obj_type}-obj3", 1000)
+            else:
+                resp1 = self.s3mp_obj.simple_multipart_upload("src-bucket1", f"{obj_type}-obj1",
+                                                              512, self.file_path, 4)
+                resp2 = self.s3mp_obj.simple_multipart_upload("dest-bucket1", f"{obj_type}-obj3",
+                                                              2048, self.file_path, 4)
+            LOGGER.info("Uploaded %s-obj1 to src-bucket1", obj_type)
+            LOGGER.info("Uploaded %s-obj3 to dest-bucket1", obj_type)
+            try:
+                self.multi_parallel_copy_object(("src-bucket1", f"{obj_type}-obj1", "dest-bucket1",
+                                                f"{obj_type}-obj2"),
+                                                ("dest-bucket1", f"{obj_type}-obj2", "src-bucket1",
+                                                f"{obj_type}-obj1"),
+                                                numproc=2,
+                                                assert_flag=False)
+            except CTException as error:
+                LOGGER.info(error.message)
+                assert_utils.assert_in(errmsg.NO_SUCH_KEY_ERR,error.message, error)
+            LOGGER.info("List %s-obj2 to dest-bucket1", obj_type)
```

Comment:
-obj2 only? because in line 455 you are listing obj1

Reply:
added log